go/utils/checkcommitters/main.go

```diff
@@ -55,23 +55,65 @@ var AllowedCommitters = map[string]*struct{}{
 // - This would be a place to enforce DCO or CLA requirements.
```

Comment:
Should we just turn on DCO bot? I did so for our fork of go-mysql-server, it's a one-click deal. It would mean anyone with outstanding commits would need to rebase.

Reply:
That's fine too, but if we enforce DCO, using this in prepr.sh is still nice?